### PR TITLE
Track embezzlers fought throughout the day

### LINKED ISSUE #258: Feature Request: Unified Log
We should have a property or file that can contain lots of information about the current session, to be printed at the end of a successful run, e.g.

* How many KGE we fought
* How many digitized combats we did
* What our average MPA was (Both considering and not KGE)
* What Guzzlr quests we did
* How many turns until the guzzlr quest is complete
* etc.

### PR DESCRIPTION
**Pull request by horrible-little-slime**
Partially resolves #258

### CODE REVIEW

File: src/fights.ts [hunk 1]
```diff
@@ -529,6 +536,12 @@ export function dailyFights(): void {
             }
           }
         });
+        if (
```
Comment (phulin) [inline src/fights.ts:539]: add braces please

File: src/index.ts [hunk 2]
```diff
@@ -207,14 +209,22 @@ function barfTurn() {
...
   ascending: false,
 };
 
-export function canContinue(): boolean {
+export const log = {
+  embezzlersFought: 0,
+  digitizedEmbezzlersFought: 0,
+};
+
+export function canContinue() {
```
Comment (phulin) [inline src/index.ts:227]:
```suggestion
export function canContinue(): boolean {
```

**Review by Tokoeka — APPROVED** [review]

File: src/fights.ts [hunk 3]
```diff
@@ -96,6 +97,7 @@ import {
 } from "./outfit";
 import { withStash } from "./clan";
 import { withChoice, withChoices } from "libram/dist/property";
+import { log } from ".";
```
Comment (phulin) [inline src/fights.ts:100]: can you put log in its own file? Don't like importing stuff from index

File: src/index.ts [hunk 4]
```diff
@@ -207,13 +209,21 @@ function barfTurn() {
...
+    else log.embezzlersFought++;
 }
 
 export const globalOptions: { ascending: boolean; stopTurncount: number | null } = {
   stopTurncount: null,
   ascending: false,
 };
 
+export const log = {
+  embezzlersFought: 0,
```
Comment (phulin) [inline src/index.ts:223]: Can we rename to initialEmbezzlersFought as that's what this is

File: src/index.ts [hunk 5]
```diff
@@ -112,6 +113,7 @@ function dailySetup() {
 }
 
 function barfTurn() {
+  const startTurns = totalTurnsPlayed();
```
Comment (phulin) [inline src/index.ts:116]: can we rename to beforeAdventureTurnsPlayed or similar more descriptive name

File: src/fights.ts (hunk 3 above)
Comment (horrible-little-slime) [reply src/fights.ts:100]: That seems reasonable. I'll move globalOptions as well

**Comment by phulin** [comment]
Needs rebase, sorry

**Pull request merged**